src/lib.rs

```diff
@@ -38,6 +38,8 @@
 #![deny(unused_must_use)]
 #![deny(broken_intra_doc_links)]
 
+pub extern crate core;
```

Comment:
I'm really not a huge fan of re-exporting `core` as `crate::core`, do we need to re-export it, and if so, can we do it inside a `#[doc(hidden)]` block with a different name?

Reply:
Uh, right, totally missed that `pub`. Please avoid doing so if possible, it messes up my auto-imports :laughing:

Reply:
I missed that too, it was in the original PR.